js/src/Components/Widgets/AutocompleteReferenceWidget.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {FormHelperText} from "@material-ui/core";
+import TextField from '@material-ui/core/TextField';
+import FormGroup from '@material-ui/core/FormGroup';
+import Autocomplete from '@material-ui/lab/Autocomplete';
```

Comment:
Handy

Reply:
> The default values are the defaults as defined in the field settings right? I didn't catch those. Most of our widgets currently don't do that either. Should we create a separate ticket to work on each of those?

Yeah, I think a separate issue is the right way to go. I don't think it is working anywhere yet.